src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -236,8 +239,9 @@ def get_detection_settings(project_id: Optional[int] = None) -> Dict[DetectorTyp
         DetectorType.CONSECUTIVE_HTTP_OP: {
             "span_duration_threshold": 1000,  # ms
             "consecutive_count_threshold": 3,
-            "max_duration_between_spans": 10000,  # ms
+            "max_duration_between_spans": 5000,  # ms
```

Comment:
<1000ms even <100ms, or maybe this should be another option, 10s is waaaaaay to long on the frontend, 1 second is still very long... I don't know about this.

Reply:
Agreed, 5s still felt very long but I wasn't sure how aggressive to be here. If you're cool with it I can just turn this into an option the same way I did for the LCP threshold and we can try that.

Also, this becomes less meaningful on the frontend because we're bounded by LCP (imo, any consecutive HTTP requests that occur before an LCP should probably get investigated) and it mostly applies to the backend. Does that change how we view if this 5s threshold is reasonable or not? I still think we could decrease it further though

Reply:
It's still meaningful on the frontend in that the time between spans is usually the render time before a new request is kicked off, with >100ms it's possible they are completely unrelated spans. Just turn it into an option we can see if the effects when changing the values.